### PR TITLE
Add Cosium

### PR DESCRIPTION
**Pull request by reda-alaoui**
## Add/Update/Remove <CompanyName>

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ Also check out [No Whiteboards](https://www.nowhiteboard.org) to search for jobs
 - [Convious](https://www.convious.com) | Vilnius, Lithuania | Intro interview (video call), followed by a homework assignment, followed by a bit more technical interview discussing homework, technical background, etc. Homework feedback is always provided.
 - [Cookpad](https://cookpad.com/us) | Tokyo, Japan; Bristol, UK | Interviews, discussion of technical background and experiences, remotely pair with devs.
 - [Coorp Academy](https://www.coorpacademy.com) | Paris, France | Technical interview as an open discussion
+- [Cosium](https://www.cosium.com) | Versailles, France | Phone screen, take-home coding challenge, remote or on-site technical interview
```
Comment (cherscarlett) [inline README.md:220]: Can you explain what the technical interview entails?

**Comment by reda-alaoui** [comment]
Ping :)

**Comment by poteto** [comment]
@reda-alaoui are any of the questions asked derived from LeetCode-style interview questions?

**Comment by reda-alaoui** [comment]
@poteto No. We have only questions about the language fundamentals. 
e.g. for `java` : what does `Stream.flatMap` do ?

**Comment by poteto** [comment]
@reda-alaoui Thanks! There's a merge conflict btw

**Comment by reda-alaoui** [comment]
@poteto , merge conflict fixed :+1:

**Comment by poteto** [comment]
@reda-alaoui thanks!

bors r+

**Pull request merged**